avro_schema/frontend.lua

```diff
@@ -607,7 +631,9 @@ local function postprocess_copy_schema(context)
 end
 
 -- validate schema definition (creates a copy)
+local cs_prepopulated_stack = utils.init_fstack({'node'})
 local function create_schema(schema, options)
+    cs_prepopulated_stack:clear()
```

Comment:
I thought about it. In that case, we need to reraise an error.
I do not like that, because it is bad for debugging.
My opinion: performance is not critical for this function, so, we can call this cheap method two times.

Reply:
Ok, got it.